wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@JonWel : about the buffer - it's up to you, but then you are in charge for guaranteeing that there will be no possibility that two functions will try to use it at the same time (like writing into this buffer a string that is based on this buffer's contents from other function). AFAICT, it doesn't seem to be a problem unless it's not thread-safe. Are Julia's global variables thread-local?